packages/@aws-cdk/aws-redshift/lib/private/database-query.ts

```diff
@@ -34,7 +35,7 @@ export class DatabaseQuery<HandlerProps> extends Construct implements iam.IGrant
       code: lambda.Code.fromAsset(path.join(__dirname, 'database-query-provider'), {
         exclude: ['*.ts'],
       }),
-      runtime: lambda.Runtime.NODEJS_14_X,
+      runtime: AwsCustomResource.regionalDefaultRuntime(this),
```

Comment:
Is it on `AwsCustomResource` or on `CustomResourceProvider` ?

Honestly, I'm fine with you sticking it somewhere in `core` or `lambda`. Maybe as a global function, so it doesn't appear in jsii?

Reply:
So both, because there are two different types here unfortunately. We wrap the runtime strings in a separate type for CustomResourceProviderRuntime 😞